sonar-python-plugin/src/test/java/org/sonar/plugins/python/SonarQubePythonFileTest.java

```diff
@@ -20,24 +20,87 @@
 package org.sonar.plugins.python;
 
 import java.io.File;
+import java.io.FileNotFoundException;
 import java.nio.charset.StandardCharsets;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
+import org.sonar.api.SonarQubeSide;
 import org.sonar.api.batch.fs.InputFile;
-import org.sonar.api.batch.fs.internal.DefaultInputFile;
+import org.sonar.api.batch.fs.internal.DefaultFileSystem;
+import org.sonar.api.batch.sensor.internal.SensorContextTester;
+import org.sonar.api.internal.SonarRuntimeImpl;
+import org.sonar.api.utils.Version;
+import org.sonar.python.PythonFile;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class SonarQubePythonFileTest {
 
+  private File baseDir = new File("src/test/resources/org/sonar/plugins/python");
+  private File file = new File(baseDir, "SonarQubePythonFile.txt");
+  private File unkownFile = new File(baseDir, "xxx");
+  private SensorContextTester context = SensorContextTester.create(baseDir);
+  private InputFile inputFile = mock(InputFile.class);
+
   @Rule
   public ExpectedException thrown = ExpectedException.none();
 
   @Test
-  public void unknown_file() throws Exception {
-    InputFile inputFile = new DefaultInputFile("key", "xxx").setModuleBaseDir(new File(".").toPath());
-    SonarQubePythonFile pythonFile = new SonarQubePythonFile(inputFile, StandardCharsets.UTF_8);
+  public void sq62() throws Exception {
+    setRuntime(Version.create(6, 2));
+    when(inputFile.contents()).thenReturn("Hello 6.2!");
```

Comment:
@ivandalbosco I believe what we currently test is right. For 6.2, we don't want to check that PythonFile.content() returns the actual content of the file, we want to check that it returns what InputFile.contents() returns. That's the crucial thing: we want to check that we don't read the file directly.